md/docs/derby-0.10/views/template-syntax/view-attributes.md

```diff
@@ -112,6 +115,14 @@ Usage:
 <tabs selected-index="{{currentTab}}" panes="{{paneOptions}}"></tabs>
 
 <tabs>
+  <!-- An example of how views can be passed in via custom tabs. -->
+  <list-icon>
+    <div class="circle"></div>
+  </list-icon>
+  <!--
+    An example of how data can be passed in via custom tabs. Directly passing in
+    data through an HTML attribute is preferred over this.
+  -->
```

Comment:
This could be more clear, suggestions? What I mean by this is:

`<view is="blah" customAttribute="{{foo}}"/>` is preferable to `<view is "blah"><custom-attribute>{{foo}}</custom-attribute></view>` because it is more explicit (that we want to pass in data, not a view), and reduces the overhead that Derby needs to figure this out.